source/documentation/disaster_recovery_guide/task/task_CreateCleanupPlaybook.adoc

```diff
@@ -1,12 +1,12 @@
 [[create_cleanup]]
-==== Create the Playbook to Clean the Primary Site
+= Create the Playbook to Clean the Primary Site
 
-Before you failback to the primary site, you need to ensure that the primary site is cleaned of all storage domains to be imported. This can be performed manually on the {engine-name}, or optionally you can create an Ansible playbook to perform it for you.
+Before you failback to the primary site, you must ensure that the primary site is cleaned of all storage domains to be imported. You can do this manually on the {engine-name}, or optionally you can create an Ansible playbook to do it for you.
```

Comment:
Although I'm not a native English speaker, but IMHO "to do it for you" should be:
"to perform it for you"
or
"to execute it for you"

Reply:
I actually was trying to make it simper language, and also to keep it parallel with the "do" earlier in the sentence:

"You can *do* this manually" ... "to *do* it for you."

Using "perform/execute" in one, but not both of these cases would, at least for me, make the sentence less clear (using the same word to refer to the same thing is a rule in technical writing). And for me, using "do" is just as clear as "perform" or "execute" and is smoother and less stiff. When I try "perform" or "execute", it almost gets in my way. This is very subjective, of course.